download_lean.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import tarfile, subprocess, sys
+
+lean_URL = "https://github.com/leanprover/lean/releases/download/v3.2.0/lean-3.2.0-linux.tar.gz"
```

Comment:
I think there are several available options:

1. Fix a version number or a git commit hash, but specify the number only once in the header, for the rest of the code should not depend on its value.

2. Get the latest version (released or on GitHub) every time, but clearly print out the version so that we can reliably debug by looking at the Travis log.